Lombiq.Tests.UI/Extensions/BasicOrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -0,0 +1,188 @@
+using Atata;
+using Lombiq.Tests.UI.Constants;
+using Lombiq.Tests.UI.Models;
+using Lombiq.Tests.UI.Pages;
+using Lombiq.Tests.UI.Services;
+using Shouldly;
+using System;
+
+namespace Lombiq.Tests.UI.Extensions
+{
+    public static class BasicOrchardFeaturesTestingUITestContextExtensions
+    {
+        public static UITestContext TestBasicOrchardFeatures(this UITestContext context) =>
+            context
+                .TestSetupWithInvalidData()
+                .TestSetup()
+                .TestRegistrationWithInvalidData()
+                .TestRegistration()
+                .TestRegistrationWithAlreadyRegisteredEmail()
+                .TestLoginWithInvalidData()
+                .TestLogin()
+                .TestTurningFeatureOnAndOff()
+                .TestLogout();
+
+        public static UITestContext TestSetup(this UITestContext context, OrchardCoreSetupParameters parameters = null)
+        {
+            parameters ??= new OrchardCoreSetupParameters
+            {
+                RecipeId = "Lombiq.OSOCE.BasicOrchardFeaturesTests",
+            };
+
+            return context.ExecuteTest(
+                "Test setup",
+                () =>
+                context.GoToSetupPage()
+                    .SetupOrchardCore(parameters)
+                    .ShouldLeaveSetupPage());
```

Comment:
Similarly in the other cases too:

```suggestion
                () => context
                    .GoToSetupPage()
                    .SetupOrchardCore(parameters)
                    .ShouldLeaveSetupPage());
```